content/blog/2019-12-02-equivalence-modulo-inputs/index.md

```diff
@@ -0,0 +1,400 @@
++++
+title = "Compiler Validation via Equivalence Modulo Inputs"
+[extra]
+bio = """
+  [Rolph Recto](rolph-recto.github.io) is a third-year graduate student studying
```

Comment:
You probably want a protocol prefix on that URL, lest browsers interpret it as a relative path.